presto-docs/src/main/sphinx/admin/queue.rst

```diff
@@ -8,11 +8,17 @@ The filename of the JSON config file should be specified in ``query.queue-config
 config property.
 
 Rules that specify multiple queues will cause the query to enter the queues sequentially.
-Rules are processed sequentially and the first one that matches will be used.
-In the example configuration below, there are five queue templates. In the
-``user.${USER}`` queue, ``${USER}`` will be expanded to the name of the user
-that submitted the query. ``${SOURCE}`` is also supported, which expands to the
-source submitting the query.
+The query must enter all rule's queues before it starts being executed. It enters the
```

Comment:
It might be better to describe this in terms of "acquiring permits", which is an easier concept to grasp than "a query is in multiple queues".